content/id/docs/tasks/tools/install-kubectl.md

```diff
@@ -0,0 +1,496 @@
+
+Untuk memastikan, muat ulang terminal kamu dan jalankan `type _init_completion`. Jika perintah berhasil maka instalasi selesai. Jika tidak, tambahkan teks berikut ke dalam _file_ `~/.bashrc`:
+
+```shell
+source /usr/share/bash-completion/bash_completion
+```
+
+Muat ulang lagi terminalmu dan pastikan bash-completion sudah berhasil diinstal dengan menjalankan `type _init_completion`.
+
+### Menyalakan _auto complete_ kubectl
+
+Sekarang kamu harus memastikan bahwa _completion script_ untuk `kubectl` sudah dimasukkan sebagai sumber _auto complete_ di semua sesi terminal. Kamu dapat melakukannya dengan dua cara:
+
+- Masukkan _completion script_ sebagai sumber di _file_ `~/.bashrc`:
+
+    ```shell
+    echo 'source <(kubectl completion bash)' >>~/.bashrc
+    ```
+
+- Menambahkan _completion script_ ke direktori `/etc/bash_completion.d`:
+
+    ```shell
+    kubectl completion bash >/etc/bash_completion.d/kubectl
+    ```
+
+Jika kamu menggunakan alias untuk `kubectl`, kamu masih dapat menggunakan fitur _auto complete_ dengan menjalankan perintah:
+
+    ```shell
+    echo 'alias k=kubectl' >>~/.bashrc
+    echo 'complete -F __start_kubectl k' >>~/.bashrc
+    ```
+
+{{< note >}}
+Semua sumber _completion script_ bash-completion terdapat di `/etc/bash_completion.d`.
+{{< /note >}}
+
+Kedua cara tersebut sama, kamu bisa mengambil salah satu cara saja. Setelah memuat ulang terminal, _auto complete_ dari `kubectl` seharusnya sudah dapat bekerja.
+
+{{% /tab %}}
+
+
+{{% tab name="Bash di macOS" %}}
+
+
+### Pendahuluan
+
+_Completion script_ `kubectl` untuk Bash dapat dibuat dengan perintah `kubectl completion bash`. Masukkan skrip tersebut ke dalam terminal sebagai sumber untuk menyalakan _auto complete_ dari `kubectl`.
+
+Namun, _completion script_ tersebut tergantung dengan [**bash-completion**](https://github.com/scop/bash-completion), yang artinya kamu harus menginstal program tersebut terlebih dahulu.
+
+{{< warning>}}
+Terdapat dua versi bash-completion, v1 dan v2. V1 untuk Bash 3.2 (_default_ dari macOs), dan v2 untuk Bash 4.1+. _Completion script_ `kubectl` **tidak kompatibel** dengan bash-completion v1 dan Bash 3.2. Dibutuhkan **bash-completion v2** dan **Bash 4.1+** agar _completion script_ `kubectl` dapat bekerja dengan baik. Maka dari itu, kamu harus menginstal dan menggunakan Bash 4.1+ ([*panduan*](https://itnext.io/upgrading-bash-on-macos-7138bd1066ba)) untuk dapat menggunakan fitur _auto complete_ dari `kubectl`. Ikuti panduan di bawah setelah kamu menginstal Bash 4.1+ (yang artinya Bash versi 4.1 atau lebih baru).
+{{< /warning >}}
+
+### Pembaruan Bash
+
+Panduan di bawah berasumsi kamu menggunakan Bash 4.1+. Kamu dapat memeriksa versi Bash dengan menjalankan:
+
+```shell
+echo $BASH_VERSION
+```
+
+Jika versinya sudah terlalu usang, kamu dapat menginstal/membaruinya dengan menggunakan Homebrew:
```

Comment:
nit: memperbarui
```suggestion
Jika versinya sudah terlalu usang, kamu dapat menginstal/memperbarui-nya dengan menggunakan Homebrew:
```

Reply:
should we put `-`? how about just `memperbaruinya`? @girikuncoro

Reply:
sure that works as well